src/components/NftImageGallery/NftImageGallery.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import {storiesOf} from '@storybook/react-native'
 import React from 'react'
 
-import {SkeletonGallery} from './ImageGallery'
+import {SkeletonGallery} from './NftImageGallery'
 
 storiesOf('NFT/GallerySkeleton', module).add('Skeleton', () => <SkeletonGallery amount={3} />)
```

Comment:
It is ok if we cover this in a different ticket? This implementation was added for the NFT gallery epic.

Let me know if you think otherwise, please.